pkg/api/postage.go

```diff
@@ -166,3 +166,55 @@ func (s *server) postageGetStampHandler(w http.ResponseWriter, r *http.Request)
 	}
 	jsonhttp.OK(w, &resp)
 }
+
+func (s *server) postageTopUpHandler(w http.ResponseWriter, r *http.Request) {
+	idStr := mux.Vars(r)["id"]
```

Comment:
once this is moved to the debugapi, you;ll need to acquire the semaphore protecting multiple postage contract events